examples/step-64/step-64.cu

```diff
@@ -236,17 +270,22 @@ namespace Step64
     const QGauss<1> quad(fe_degree + 1);
     mf_data.reinit(mapping, dof_handler, constraints, quad, additional_data);
 
-    // We need to store the value of the coefficient for each quadrature point
-    // in every locally owned cells.
+
     const unsigned int n_owned_cells =
-      std::distance<decltype(dof_handler.begin_active())>(
-        dof_handler.begin_active(), dof_handler.end());
+      dynamic_cast<const parallel::Triangulation<dim> *>(
+        &dof_handler.get_triangulation())
+        ->n_locally_owned_active_cells();
     coef.reinit(Utilities::pow(fe_degree + 1, dim) * n_owned_cells);
-    VaryingCoefficientFunctor<dim, fe_degree> functor(coef.get_values());
+
+    const VaryingCoefficientFunctor<dim, fe_degree> functor(coef.get_values());
```

Comment:
The `const` is a bit counter-intuitive since we are modifying the values stored in the pointer member variable in the next line. I am not sure if it's better to keep `const` or not.

Reply:
I don't feel strongly about it. I think the question is: Are we modifying the *functor*, or are we modifying the stored coefficient values? The former expires at the end of the function, the latter does not.

Reply:
I am fine with this version.